src/__mocks__/aws-sdk.ts

```diff
@@ -17,6 +17,14 @@ class CloudFormation {
       })
     };
   }
+
+  deleteStack(_params) {}
 }
 
-module.exports = { CloudFormation };
+const DynamoDB = {
+  DocumentClient: class {
```

Comment:
Is there a reason we're going down the path of OOP?

Reply:
Just following actual aws-sdk implementation for CloudFormation class, otherwise mocks does not work.
Also, If you look at, `src/logCloudFormationStack.ts:7`
`const cloudFormation = new CloudFormation();`

This is expecting a class.

However,  DynamoDB is an Object 
`const documentClient = new DynamoDB.DocumentClient();`

Hope it makes sense. Feel free to advise patterns